pyquil/device/_library.py

```diff
@@ -0,0 +1,50 @@
+import networkx as nx
+from . import Device
+from ._specs import Specs, specs_from_graph
+from ._isa import ISA, isa_from_graph, isa_from_digraph
+
+
+def ibmq_ourense(oneq_type='Xhalves', twoq_type='CNOT'):
+    graph = nx.from_edgelist([
+        (0, 1), (1, 0),
+        (1, 2), (2, 1),
+        (1, 3), (3, 1),
+        (3, 4), (4, 3)
+    ])
+
+    return Device(name="ibmq_ourense",
+                  raw={"isa": isa_from_digraph(graph, oneq_type=oneq_type, twoq_type=twoq_type).to_dict(),
+                       "specs": specs_from_graph(graph).to_dict()})
+
+
+def ibmq_yorktown(oneq_type='Xhalves', twoq_type='CNOT'):
```

Comment:
so IBM does this U1/U2/U3 thing for their gates -- could we replicate that here? if I understand correctly

- `U1(a)` is just `RZ(a)`
- `U2(a,b)` is `RZ(b)`, `RX(pi/2)`, `RZ(a)`
- `U3(a,b,c)` is  `RZ(c)`, `RX(pi/2)`, `RZ(b)`, `RX(pi/2)`, `RZ(a)`

might need to be implemented at the quilc level / be a future PR but just wanted to mention it